README.md

```diff
@@ -144,7 +169,7 @@ _FieldDB_ is actively worked on by the [https://github.com/OpenSourceFieldlingui
 * [UsageDashboard](https://github.com/OpenSourceFieldlinguistics/UsageDashboard) A realtime updating usage dashboard for fielddb servers (forked from dashing) (_Fork_)
 
 ## Modules
-
+_the term "modules" means something in Python. What does it mean here?_
```

Comment:
Can you also remove this inline comment? Comments should only be in the issues. Thanks.